src/struct_mv/struct_matmult_fuse.c

```diff
@@ -4693,11 +4693,6 @@ hypre_StructMatmultCompute_fuse( HYPRE_Int nterms,
    /* Call individual fuse functions */
    if (!combined_fcc_fc_f)
    {
-      /* First, we need to initialize Mptrs to zeroes */
-      hypre_Memset((void *) hypre_StructMatrixVData(M), 0.0,
-                   sizeof(HYPRE_Complex) * (Mdata_size - Mvdata_offset),
-                   hypre_StructMatrixMemoryLocation(M));
-
```

Comment:
Why was this memset in here in the first place? Is it OK to just remove it?

Reply:
This one is fine removing because now we are doing `CTAlloc` always regardless of the value of `zero_init`. It was here in the first place because the kernels below assume that `hypre_StructMatrixVData(M)` is initialized

Reply:
OK. Do you want to comment it out instead and mark it with a TODO for when we revisit the zero_init feature? Up to you.